met-web/package.json

```diff
@@ -37,14 +39,16 @@
         "react-dom": "^18.0.0",
         "react-draft-wysiwyg": "^1.14.7",
         "react-dropzone": "^14.2.1",
+        "react-hook-form": "^7.37.0",
         "react-if": "^4.1.4",
         "react-redux": "^7.2.8",
         "react-router-dom": "^6.3.0",
         "react-scripts": "5.0.1",
         "react-svg": "^15.0.1",
         "redux": "^4.1.2",
         "typescript": "^4.6.3",
-        "web-vitals": "^2.1.4"
+        "web-vitals": "^2.1.4",
+        "yup": "^0.32.11"
```

Comment:
Its a new library for me..I have used some other libraries for validation ;but not this one
Hope you have checked licence, stars ,frequency of release etc..Let me know if you want to discuss..

Reply:
Yes, it checks out. It's very popular, well maintained library with MIT license
![image](https://user-images.githubusercontent.com/91914654/196819857-bfb8786a-b87e-4715-bb74-b01da5e179b4.png)

Reply:
![image](https://user-images.githubusercontent.com/91914654/196822711-6b3e05a5-b016-4482-9746-d251537a6937.png)